modules/support-insecure-tls-connections.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-insecure-tls-connections_{context}"]
+= Support for insecure TLS connections
+
+If a custom CA certificate is used, then thw `must-gather` pod fails to grab the output for `velero logs/describe`. To allow insecure TLS connections, you can use the following command to pass a flag to the `must-gather` command:
+
+[source,terminal]
+----
+oc adm must-gather --image=quay.io/konveyor/oadp-must-gather:latest -- /usr/bin/gather_without_tls <true/false>
```

Comment:
```suggestion
oc adm must-gather --image=registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.3 -- /usr/bin/gather_without_tls <true/false>
```

Any way to have this be a variable?

The image used should be the same one used in https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/troubleshooting.html#migration-using-must-gather_oadp-troubleshooting section.

Which is ideally specific image to the version of operator installed.

Reply:
@kaovilai @anarnold97  

1.  I can easily create a variable , but I see in the documentation that we have an image specified for 1.2 and 1.3 and I'd guess we'll have one for 1.4. Therefore, I suggest adding one variable now called "mustgather-image-1.3" (value=rhel9:v1.3), adding another called "mustgather-image-1.4" when the time comes, and leaving the image for 1.2 hard-coded, as is, since 1.2's days are numbered .   WDYT?

Reply:
@kaovilai For now, I've made the change without using a new attribute.

Reply:
Works